coil-base/src/androidTest/java/coil/fetch/UriFetcherTest.kt

```diff
@@ -88,4 +87,29 @@ class UriFetcherTest {
         val result = loader.extractAssetFileName(uri)
         assertEquals(null, result)
     }
+
+    @Test
+    fun fileCacheKeyWithLastModified() {
+        val file = File(context.filesDir.absolutePath + File.separator + "file.jpg")
+        val uri = Uri.fromFile(file)
```

Comment:
Nit: Can you swap this to `file.toUri()`?

Reply:
done.